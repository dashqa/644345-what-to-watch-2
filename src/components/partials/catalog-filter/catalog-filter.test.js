import React from 'react';
import renderer from 'react-test-renderer';
import CatalogFilter from './catalog-filter';

const props = {
  active: `Comedy`,
  genres: new Set([`Comedy`]),
  onChange: () => {}
};

it(`Catalog filter correctly renders after relaunch`, () => {
  const tree = renderer.create(<CatalogFilter {...props}/>).toJSON();
  expect(tree).toMatchSnapshot();
});
