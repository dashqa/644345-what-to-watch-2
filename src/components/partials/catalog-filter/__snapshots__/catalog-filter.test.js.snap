// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Catalog filter correctly renders after relaunch 1`] = `
<ul
  className="catalog__genres-list"
>
  <li
    className="catalog__genres-item"
    onClick={[Function]}
  >
    <a
      className="catalog__genres-link"
    >
      comedy
    </a>
  </li>
</ul>
`;
